### PR TITLE
Intly 7381 - take 2

### PR DESCRIPTION
**Pull request by briandooley**

### CODE REVIEW

**Comment by nebelsprachen** [comment]
LGTM

**Review by nebelsprachen — APPROVED** [review]

File: modules/ROOT/pages/_partials/rn-new-and-changed-ref.adoc [hunk 1]
```diff
@@ -1,52 +1,15 @@
...
-|{three-scale-name}
-|link:{three-scale-rn-docs}[{three-scale-version}]
-
-|{code-ready-name}
-|link:{code-ready-rn-docs}[{code-ready-version}]
-
-|===
-
-
-[id='configuring-amq']
```
Comment (pwright) [inline modules/ROOT/pages/_partials/rn-new-and-changed-ref.adoc:47]: if you remove this, you break the link to it @briandooley

Comment (briandooley) [reply modules/ROOT/pages/_partials/rn-new-and-changed-ref.adoc:47]: The link is no longer necessary. The text of the section is now included in the list of new and changed features and is no longer its own section.

image

Comment (briandooley) [reply modules/ROOT/pages/_partials/rn-new-and-changed-ref.adoc:47]: References to that should have been taken out as that section is now included in the new and changed features.  Have created [this](https://github.com/integr8ly/user-documentation/pull/131) to fix.

**Pull request merged**